scripts/make-packages.sh

```diff
@@ -139,62 +139,60 @@ echo "ARCH=$ARCH PKGARCH=$PKGARCH ARTIFACTS=$ARTIFACTS"
+  fi
+  if [ "$PKGARCH" == 'aarch64' ] || [ "$PKGARCH" == 'x86_64' ]; then
+    BITS='64bit'
+  fi
   # XDP is only validated on Ubuntu 24.04 and x64
-  if [ "$XDP" == "False" ] || [[ "$ARCH" == arm* ]]; then
-    # RedHat/CentOS
-    FILES="${ARTIFACTS}/libmsquic.${LIBEXT}.${VER_MAJOR}.${VER_MINOR}.${VER_PATCH}=/usr/${LIBDIR}/libmsquic.${LIBEXT}.${VER_MAJOR}.${VER_MINOR}.${VER_PATCH}"
-    FILES="${FILES} ${ARTIFACTS}/libmsquic.${LIBEXT}.${VER_MAJOR}=/usr/${LIBDIR}/libmsquic.${LIBEXT}.${VER_MAJOR}"
-    if [ -e "$ARTIFACTS/libmsquic.lttng.${LIBEXT}.${VER_MAJOR}.${VER_MINOR}.${VER_PATCH}" ]; then
-       FILES="${FILES} ${ARTIFACTS}/libmsquic.lttng.${LIBEXT}.${VER_MAJOR}.${VER_MINOR}.${VER_PATCH}=/usr/${LIBDIR}/libmsquic.lttng.${LIBEXT}.${VER_MAJOR}.${VER_MINOR}.${VER_PATCH}"
-    fi
-    if [ "$PKGARCH" == 'aarch64' ] || [ "$PKGARCH" == 'x86_64' ]; then
-      BITS='64bit'
-    fi
-    if [ "$XDP" == "True" ] && [[ "$ARCH" == x* ]]; then
-      echo "Building rpm package (XDP)"
-      fpm \
-        --force \
-        --input-type dir \
-        --output-type rpm \
-        --architecture ${PKGARCH} \
-        --name ${NAME} \
-        --provides ${NAME} \
-        --depends "libcrypto.so.${TLSVERSION}()(${BITS})" \
-        --depends "libnuma.so.1()(${BITS})" \
-        --depends "libxdp.so.1.4.0" \
-        --depends "libnl-route-3.so.200" \
-        --conflicts ${CONFLICTS} \
-        --version ${VER_MAJOR}.${VER_MINOR}.${VER_PATCH} \
-        --description "${DESCRIPTION}" \
-        --vendor "${VENDOR}" \
-        --maintainer "${MAINTAINER}" \
-        --package "${OUTPUT}" \
-        --license MIT \
-        --url https://github.com/microsoft/msquic \
-        --log error \
-        ${FILES} ${ARTIFACTS}/datapath_raw_xdp_kern.o=/usr/${LIBDIR}/datapath_raw_xdp_kern.o
-    else
-      echo "Building rpm package"
-      fpm \
-        --force \
-        --input-type dir \
-        --output-type rpm \
-        --architecture ${PKGARCH} \
-        --name ${NAME} \
-        --provides ${NAME} \
-        --depends "libcrypto.so.${TLSVERSION}()(${BITS})" \
-        --depends "libnuma.so.1()(${BITS})" \
-        --conflicts ${CONFLICTS} \
-        --version ${VER_MAJOR}.${VER_MINOR}.${VER_PATCH} \
-        --description "${DESCRIPTION}" \
-        --vendor "${VENDOR}" \
-        --maintainer "${MAINTAINER}" \
-        --package "${OUTPUT}" \
-        --license MIT \
-        --url https://github.com/microsoft/msquic \
-        --log error \
-        ${FILES}
-    fi
+  if [ "$XDP" == "True" ] && [[ "$ARCH" == x* ]]; then
+    echo "Building rpm package (XDP)"
+    fpm \
+      --force \
+      --input-type dir \
+      --output-type rpm \
+      --architecture ${PKGARCH} \
+      --name ${NAME} \
+      --provides ${NAME} \
+      --depends "libcrypto.so.${TLSVERSION}()(${BITS})" \
+      --depends "libnuma.so.1()(${BITS})" \
+      --depends "libxdp.so.1.4.0" \
```

Comment:
Why there is a hard dependency on 1.4 version of libxdp?

Reply:
I don't have any other environment to check the libraries. at least x64 depends on it

Reply:
Yes, but I believe this should depend on 'libxdp.so.1', right? It will be symbolic link to latest version of libxdp anyway

Reply:
I can finally check on azure linux 3.0 with this change. wait a sec